functions/BBoxCombFunction.cc

```diff
@@ -0,0 +1,136 @@
+// -*- mode: c++; c-basic-offset:4 -*-
+
+// This file is part of bes, A C++ implementation of the OPeNDAP
+// Hyrax data server
+
+// Copyright (c) 2015 OPeNDAP, Inc.
+// Authors: James Gallagher <jgallagher@opendap.org>
+//
+// This library is free software; you can redistribute it and/or
+// modify it under the terms of the GNU Lesser General Public
+// License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+//
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public
+// License along with this library; if not, write to the Free Software
+// Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+//
+// You can contact OPeNDAP, Inc. at PO Box 112, Saunderstown, RI. 02874-0112.
+
+#include "config.h"
+
+#include <cassert>
+#include <sstream>
+#include <memory>
+
+#include <BaseType.h>
+#include <Int32.h>
+#include <Str.h>
+#include <Array.h>
+#include <Structure.h>
+
+#include <D4RValue.h>
+#include <Error.h>
+#include <debug.h>
+#include <util.h>
+#include <ServerFunctionsList.h>
+
+#include "BBoxCombFunction.h"
+#include "roi_util.h"
+
+using namespace std;
+using namespace libdap;
+
+namespace functions {
+
+/**
+ * @brief Combine two bounding boxes with different shapes, forming their combination.
+ *
+ * This combines 2 BBox variables (Array of Structure) forming
```

Comment:
Change to:
This combines to bounding boxes (Array of Structure) of rank N and M into a single bounding box of rank N+M.